### ISSUE TITLE
Installation script for Centos9 and Fedora 36

### ISSUE THREAD

**Issue by LecrisUT** [zerotier/ZeroTierOne#1655]
There is a minor issue with the installation when run on `CentOS9`, i.e. a symlink does not exist at `http://download.zerotier.com/redhat/el/9/repodata/repomd.xml`.

But the main issue for both is that zerotier built there requires OpenSSL 1.1 (i.e. `/usr/sbin/zerotier-one: error while loading shared libraries: libssl.so.1.1`), although that is not included in either of these distros (`3.0`). There are `compat-openssl11` pacakges available to make zerotier run on these distros, but these need to be included in the installation script. But alternatively, when will ZeroTier upgrade to openssl 3.0?

**Comment by erikh** [comment]
So, this is basically the same as #1652; the issue is with the missing dep on libssl3. We're preparing packages which should be ready in the next few days.

**Comment by LecrisUT** [comment]
Indeed, but don't forget the missing symlink for RHEL9. It would be preferable to have it on copr or eventually on the main Fedora repository so that it can be tested for the next releases.

**Comment by jguizarj** [comment]
Hi! any update on Fedora 36?

**Comment by someara** [comment]
Hello! We're actively working on the packaging situaiton. Stay tuned.

**Comment by goshansp** [comment]
Workaround: Fedora 36 `dnf install openssl1.1` and for Fedora 36 Silverblue`rpm-ostree install openssl1.1` does work for me.

**Comment by LecrisUT** [comment]
Is there a `compat-openssl11` package instead? Does `openssl1.1` package conflict with 3.x one?

**Comment by goshansp** [comment]
no, there is no `compat-openssl11` as mentioned in https://bugzilla.redhat.com/show_bug.cgi?id=1879067 and from my understanding fedora packages do not conflict. i have 11 and 3 installed alongside.

**Comment by LecrisUT** [comment]
Tip because a previous zerotier installation script was broken for fedora:
delete the old zerotier package and rerun the script on `install.zerotier.com`

There was an issue that zerotier was pulling `1.8.10-1.el7` instead of `1.8.10-1.el8`. But arguably it should be either `1.8.10-1.el9` or better yet `1.8.10-1.fc36`.
